spec/factories/temp_files.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
In the `spec/factories/tables.rb`, should we mock the correct `contents` contents? (it's a form of JSON)

Reply:
Hm.. I see, yes should be someting else than `Faker::Lorem.characters`. Do you have in mind any special data or just a random JSON 2dim array? Probably we are not using/testing it anywhere?